vmm/src/acpi.rs

```diff
@@ -610,14 +672,25 @@ pub fn create_acpi_tables(
             // If serial is turned off, add a fake device with invalid irq.
             31
         };
+
+        // SPCR
         let spcr = create_spcr_table(serial_device_addr, serial_device_irq);
         let spcr_offset = prev_tbl_off.checked_add(prev_tbl_len).unwrap();
         guest_mem
             .write_slice(spcr.as_slice(), spcr_offset)
             .expect("Error writing SPCR table");
         tables.push(spcr_offset.0);
-        prev_tbl_len = spcr.len() as u64;
-        prev_tbl_off = spcr_offset;
+
+        // DBG2
+        let dbg2 = create_dbg2_table(serial_device_addr);
+        let dbg2_offset = spcr_offset.checked_add(spcr.len() as u64).unwrap();
```

Comment:
I would rather you duplicate the calculation of `prev_tbl_len` and `prev_tbl_off` than to use `spcr` variables here.

Reply:
Fixed, thanks.